Utils.h

```diff
@@ -97,5 +97,22 @@ static inline void trim(std::string &s) {
     rtrim(s);
 }
 
+/**
+ * @return true if str starts with given prefix
+ */
+static bool startsWith(const std::string &str, const std::string &prefix) {
+    return str.substr(0, prefix.size()) == prefix;
+}
+
+/**
+ * @return true if checkedType uses type
+ *         example: checkedType = native.Ptr[struct_A], type = struct_A
+ */
+static bool typeUsesOtherType(const std::string &checkedType, const std::string &type) {
+    // TODO: find better way to check it
+    return checkedType == type ||
+           checkedType == "native.Ptr[" + type + "]";
```

Comment:
@jonas, yes, but there are also function pointers :( in that case it is possible to search for a substring  ", type " it doesn't seems right. 
Will it make sense to add `Type` class to IR? There will be 4 subclasses of this class:
* `PointerType`
* `ArrayType`
* `FunctionPointerType`
* `Struct/Union type` (maybe it can be just `Type`)

Types may contain other types.
There will be right way to check if a type uses another type.
Also `Type` will handle reserved words, so this part of type translation will be more clear.

Reply:
If we decide to do it then it should be separate PR. Now I will continue with this version.

Reply:
It would be nice to have a proper way to represent types. Eventually we can look into having Type reference the current Struct/Union/TypeDef classes.